gno.land/cmd/gnoland/config.go

```diff
@@ -67,24 +68,83 @@
 	return &field, nil
 }
 
-// newInvalidFieldError creates an error for non-existent struct fields
-// being passed as arguments to [getFieldAtPath]
-func newInvalidFieldError(field string, value reflect.Value) error {
-	var (
-		valueType = value.Type()
-		numFields = value.NumField()
-	)
-
-	fields := make([]string, 0, numFields)
+func fieldByTOMLName(value reflect.Value, name string) (dst reflect.Value) {
```

Comment:
I am really not a fan of names returns, and think we should avoid them entirely in the codebase 

What do you think about dropping it here?

Reply:
I have no strong feelings either way, I thought it was nice here as it avoided a declaration :)

Removing the named return